scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/controllers/TasksController.java

```diff
@@ -47,7 +47,11 @@ private GetTasksResponse from(Map.Entry<String, Task> task) {
     }
 
     private String toFormattedAddress(InetSocketAddress clientAddress) {
-        return String.format("%s:%s", clientAddress.getAddress().getHostAddress(), clientAddress.getPort());
+        if (clientAddress.isUnresolved()) { // Protect against unresolved IP addresses.
+            return "UNKNOWN";
```

Comment:
Isn't this a bit dangerous? If we're in a network without dns `"Unknown"` will be all over the place?